src/Microsoft.DocAsCode.Build.TableOfContents/BuildTocDocument.cs

```diff
@@ -99,5 +99,19 @@ private void BuildCore(TocItemViewModel item, FileModel model, IHostService host
                 }
             }
         }
+
+        #region ISupportIncrementalBuild Members
+
+        public bool CanIncrementalBuild(FileAndType fileAndType)
+        {
+            return true;
+        }
+
+        public string GetIncrementalContextHash()
+        {
+            return null;
```

Comment:
quesion: seems currently IncrementalContextHash all set to null? will we implement it later or not?